jenkinsfiles/gcp-osp-aro.Jenkinsfile

```diff
@@ -13,7 +13,7 @@ podTemplate(yaml: readTrusted('jenkinsfiles/SubmarinerAgentPod.yaml')) {
                 extendedChoice(name: 'PLATFORM', description: 'The managed clusters platform that should be tested',
                     value: 'aws,gcp,azure,vsphere,osp,aro,rosa', defaultValue: 'gcp,osp,aro', multiSelectDelimiter: ',', type: 'PT_CHECKBOX', visibleItemCount: 7),
                 booleanParam(name: 'SUBMARINER_GATEWAY_RANDOM', defaultValue: true, description: 'Deploy two submariner gateways on one of the clusters'),
-                string(name: 'NODE_TO_LABEL_AS_GW', defaultValue: 'mbabushk-osp', description: 'Specify cluster node to be manually labeled as Submariner Gateway'),
+                string(name: 'NODE_TO_LABEL_AS_GW', defaultValue: '', description: 'Specify cluster node to be manually labeled as Submariner Gateway'),
```

Comment:
Since the "NODE_TO_LABEL_AS_GW" parameter is always used for the osp based ocp cluster, and we know the name of the cluster, the default value should contains the name of the cluster so for the next execution of the job, you will not need to specify the value manually.